frontend/app/views/spree/products/_filters_desktop.html.erb

```diff
@@ -3,7 +3,7 @@
     <div class="plp-filters-scroller">
       <% @available_option_types.each do |option_type| %>
         <div class="w-100 card plp-filters-card">
-          <% ot_downcase_name = option_type.filter_param %>
```

Comment:
@aplegatt let's update the `filter_param` method here: https://github.com/spark-solutions/spree/blob/210acd9fcc143f6626ec0e4dc8133a6b2841ed27/core/app/models/spree/option_type.rb#L26